### ISSUE TITLE
Not able to use YITH WooCommerce Badge Management 

### ISSUE THREAD

**Issue by SeshiParitala** [dudaster/ele-custom-skin#214]
Can you give any custom code with a shortcode so that works for me

**Comment by dudaster** [comment]
Can’t you use Dynamic Value shortcode?

**Comment by SeshiParitala** [comment]
No, I don't have any idea on how to use a dynamic value shortcode

**Comment by dudaster** [comment]
You need to have the shortcode. And the shortcode can be called like this:
![Screen Shot 2020-05-21 at 11 37 52 AM](https://user-images.githubusercontent.com/8319178/82541290-e0d98300-9b58-11ea-89be-865c97b5439f.png)

**Comment by SeshiParitala** [comment]
Tried it not working sir

**Comment by SeshiParitala** [comment]
![E](https://user-images.githubusercontent.com/65705806/82545363-38391c80-9b74-11ea-9a95-595b91b67bb5.jpg)

**Comment by SeshiParitala** [comment]
![2](https://user-images.githubusercontent.com/65705806/82545459-5737ae80-9b74-11ea-80cd-6615cee29411.jpg)

**Comment by SeshiParitala** [comment]
On-Page its working see the URL  : http://www.mavennaturals.com/product/hyaluronic-vitamin-c-serum/

**Comment by dudaster** [comment]
Did you found a solution?

**Comment by SeshiParitala** [comment]
No, If you could help me that would be great!
